.github/workflows/update_online.yml

```diff
@@ -0,0 +1,29 @@
+name: Update the online lists
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  schedule:
+    - cron: '16 20 * * 5'
+  workflow_dispatch:
```

Comment:
Why do we need a `workflow_dispatch` for this workflow?

Reply:
The workflow_dispatch is for testing or refreshing the lists via the GitHub UI if a domain suddenly came online or something.
I can remove it

Reply:
Removed